menu/cbs/menu_cbs_title.c

```diff
@@ -1727,6 +1751,10 @@ int menu_cbs_init_bind_title(menu_file_list_cbs_t *cbs,
       {MENU_ENUM_LABEL_DEFERRED_PLAYLIST_MANAGER_SETTINGS,                                  action_get_title_deferred_playlist_list},
       {MENU_ENUM_LABEL_PLAYLISTS_TAB,                                                       action_get_title_collection},
       {MENU_ENUM_LABEL_DEFERRED_MANUAL_CONTENT_SCAN_LIST,                                   action_get_title_manual_content_scan_list},
+#ifdef HAVE_MIST
+      {MENU_ENUM_LABEL_DEFERRED_CORE_MANAGER_STEAM_LIST,                                    action_get_title_core_manager_steam_list},
+      {MENU_ENUM_LABEL_DEFERRED_CORE_INFORMATION_STEAM_LIST,                                action_get_title_core_information_steam_list},
```

Comment:
For consistency, don't bind this here - do it the same as `DEFERRED_CORE_MANAGER_STEAM_LIST`